CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Fixed
 
 - [Header.LanguageSelector] Added `aria-current` to currently selected language button
+- [Header.HeaderActionBarItemWithDropdown] Fix click event bubbling issue in arrow icon
```

Comment:
The users know this as Header.ActionBarItem. It's exported with that name, [check here](https://github.com/City-of-Helsinki/helsinki-design-system/blob/development/packages/react/src/components/header/Header.tsx#L100).

Reply:
Fixed now.